src/View/Helper/AbstractHelper.php

```diff
@@ -558,4 +559,32 @@ protected function hasAllowedPrefix(string $attribute): bool
 
         return false;
     }
+
+    /**
+     * translate the label
+     */
+    protected function translateLabel(string $label): string
+    {
+        if (null !== ($translator = $this->getTranslator())) {
+            $label = $translator->translate(
+                $label,
+                $this->getTranslatorTextDomain()
+            );
+        }
+
+        return $label;
+    }
+
+    /**
+     * escape the label
+     */
+    protected function escapeLabel(ElementInterface $element, string $label): string
+    {
+        if (! $element instanceof LabelAwareInterface || ! $element->getLabelOption('disable_html_escape')) {
+            $escapeHtmlHelper = $this->getEscapeHtmlHelper();
+            $label            = $escapeHtmlHelper($label);
```

Comment:
Return early, rather than overwriting.

Worth also inverting the conditionals to remove `!` and return early